service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataServiceTest.java

```diff
@@ -402,4 +492,30 @@ private CaseData prepareCaseDataWithOrderType(final OrderType... orderTypes) thr
                 .build())
             .build();
     }
+
+    private CaseData prepareCaseDataWithUpdatedApplicantTelephoneNumber(final String telephoneNumber)
+        throws IOException {
+        return prepareCaseData(NOW).toBuilder()
+            .applicants(wrapElements(Applicant.builder()
+                .party(ApplicantParty.builder()
+                    .telephoneNumber(Telephone.builder()
+                        .telephoneNumber(telephoneNumber)
+                        .build())
+                    .build())
+                .build()))
+            .build();
+    }
+
+    private CaseData prepareCaseDataWithUpdatedApplicantMobileNumber(final String mobileNumber)
+        throws IOException {
+        return prepareCaseData(NOW).toBuilder()
+            .applicants(wrapElements(Applicant.builder()
+                .party(ApplicantParty.builder()
+                    .mobileNumber(Telephone.builder()
+                        .telephoneNumber(mobileNumber)
+                        .build())
+                    .build())
+                .build()))
+            .build();
+    }
```

Comment:
We could just use the one method and have applicant 1 use telephone and applicant 2 mobile?